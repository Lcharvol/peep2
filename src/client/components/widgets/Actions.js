import styled from 'styled-components';

//  background: linear-gradient( to right, rgba(48, 64, 77, 0.5), rgba(48, 64, 77, 1)
const Actions = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  align-items: flex-start;
  right: 10px;
  top: 0px;
  left: 0px;
  height: 100%;
  width: 100%;
  z-index: 5;
  position: absolute;
  background: radial-gradient(
    circle at top right,
    rgba(48, 64, 77, 1),
    rgba(48, 64, 77, 0.3)
  );
`;

export default Actions;
