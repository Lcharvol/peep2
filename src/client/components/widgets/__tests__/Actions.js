import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme'; // eslint-disable-line
import Actions from '../Actions';

describe('<Actions />', () => {
  test('should render an <Action />', () => {
    const tree = shallow(<Actions />);
    expect(tree).toMatchSnapshot();
  });
  test('should <Actions /> be flex', () => {
    const tree = shallow(<Actions />);
    expect(tree).toHaveStyleRule('display', 'flex');
  });
});
