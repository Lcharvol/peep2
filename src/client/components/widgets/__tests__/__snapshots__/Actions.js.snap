// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions /> should render an <Action /> 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa eQMKpk"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa eQMKpk"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <styled.div />,
    "_instance": StyledComponent {
      "attrs": Object {},
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "generatedClassName": "eQMKpk",
        "theme": null,
      },
      "unsubscribeId": -1,
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": Object {
      "generatedClassName": "eQMKpk",
      "theme": null,
    },
    "_rendered": <div
      className="sc-bdVaJa eQMKpk"
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;
